assets/css/10ten.css

```diff
@@ -8,6 +13,23 @@ main { min-height: 80vh; }
 }
 
 .mb-10 { margin-bottom: 10em; }
+.mt-5 { margin-top: 5em; }
+.mt-2 { margin-top: 2em; }
+
+/* Reusable components */
+
+.row {
+    max-width: 1140px;
+    margin: 0 auto;
+}
+
+section {
+    padding: 10px 0;
+}
+
+.box {
+    padding: 1%;
+}
```

Comment:
Why this? Not sure if I understand...
I would add padding utility functions like we did for mt-5 and mt-2.
I would use p-1, p-2, just like bootstrap: https://getbootstrap.com/docs/5.0/utilities/spacing/

Reply:
I don't know why that entered, couldn't locate that new class so I just deleted it.